test/reducers/util.test.js

```diff
@@ -10,3 +12,35 @@ describe('toggleIncludes', () => {
     expect(toggleIncludes(['f', {x: 'g'}], {x: 'g'})).to.deep.equal(['f'])
   })
 })
+
+describe('handleSetState', () => {
+  const objectReducer = (state = {a: 1, b: 2}, action) => {
+    if (action.type === 'ADD_C') {
+      return {...state, c: action.payload}
+    }
+    return state
+  }
+  const stringReducer = (state = 'default', action) => {
+    return state
+  }
+
+  it('injects a SET_STATE clause in each reducer', () => {
```

Comment:
this name needs to be updated -- the whole test could be simplified too, no need for different types of other reducers